controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -3932,7 +3932,27 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
 
         await opts.keycloakClient.authenticateClient();
 
-        const user = await userRepo.byEmail(req.email);
+        const keycloakUser = await opts.keycloakClient.client.users.find({
+          max: 1,
+          email: req.email,
+          realm: opts.keycloakRealm,
+          exact: true,
```

Comment:
Will this also work when someone use a capitalized email?

Reply:
yes